inst/templates/ad_adlb.R

```diff
@@ -0,0 +1,328 @@
+# Name: ADLB
+#
+# Label: Lab Analysis Dataset
+#
+# Input: adsl, lb
+library(admiral)
+library(admiral.test) # Contains example datasets from the CDISC pilot project
+library(dplyr)
+library(lubridate)
+library(stringr)
+
+# ---- Load source datasets ----
+
+# Use e.g. haven::read_sas to read in .sas7bdat, or other suitable functions
+# as needed and assign to the variables below.
+# For illustration purposes read in admiral test data
+
+data("lb")
+data("adsl")
+
+lb <- convert_blanks_to_na(lb)
+
+# The CDISC Pilot Data contains no SUPPVS data
+# If you have a SUPPLB then uncomment function below
+
+# lb <- derive_vars_suppqual(lb, supplb) %>%
+
+# ---- Lookup tables ----
+adlb_dis <- adlb %>%
```

Comment:
Is this supposed to say `adlb_dis <- lb` instead? When I run this chunk of code I get an error because adlb is not created yet.

Reply:
I'm getting error as well. But since that was written from @millerg23 , I thought we could have a discussion about it later.

Reply:
Lines 29-31 should be removed. This was temporary code I used to get the list of LBTESTCD terms, for the lookup tyabel.